tests/filesystem.go

```diff
@@ -224,6 +224,7 @@ func setupDisk(ctx context.Context, disk *types.Disk, diskIndex int, imageSize i
 	loopdev := disk.Device
 	defer func() {
 		if err != nil {
+			//nolint:errcheck
 			destroyDevice(loopdev)
```

Comment:
Does something like `_ = destroyDevice(loopdev)` satisfy the linter?  That might be cleaner.

Reply:
Yeah, the linter doesn't complain after this change.